devices/Ld2410/Ld2410/Commands/EnableConfigurationCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Ld2410.Commands
+{
+    internal sealed class EnableConfigurationCommand : CommandFrame
```

Comment:
in theory, it's 1 class per file, the linter will most likely complain about it.

Reply:
Apologies as my StyleCop was not working. It is fixed now so I have corrected all style issues.